src/components/UserManagement/UserBilling/UserBilling.tsx

```diff
@@ -86,9 +90,60 @@ export default class UserBilling extends React.Component<Props, State> {
                       />
                     </button>
                   </div>
-                  {paymentMethod.isOpen && (
-                    <div className="horizontal-divider" />
-                  )}
+                  <div className={cn(styles.isMobile, styles.paymentRow)}>
```

Comment:
We could extract the paymentRows into desktop/mobile render functions, like `this.renderDesktopPaymentRow(paymentMethod, index)` and choose between them in the main render() based on isOnMobile() in order to avoid having 2 sets of elements in the DOM and just hiding one of them through CSS.